COPYING

```diff
@@ -1,4 +1,4 @@
-cats-mtl Copyright (c) 2017 Edmund Noble.
+cats-mtl Copyright (c) 2017 Cats-mtl Contributors.
```

Comment:
Do we have permission to change the copyright ownership?

Reply:
Great question, I have no idea, I just wanted to make it consistent with `cats` and `cats-effect`.

Reply:
Is that a problem? Surely it is a superset now.